data/core/tips.json

```diff
@@ -362,16 +362,146 @@
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Certain corpses can give you bionics if you dissect them." ]
+    "text": [ "Don't be too greedy.  Loot doesn't matter if you're dead." ]
   },
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "Don't be too greedy.  Loot doesn't matter if you're dead." ]
+    "text": [ "The floor is too hard to sleep on?  Try gathering a pile of leaves." ]
   },
   {
     "type": "snippet",
     "category": "tip",
-    "text": [ "The floor is too hard to sleep on?  Try gathering a pile of leaves." ]
+    "text": [ "Too cold to sleep well at night?  A pile of clothes will make it better." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Lost much blood?  Drink to replenish fluids, but red blood cells need time to renew." ]
```

Comment:
Not quite valid English. Recommend you replace with the following:

`Did you lose too much blood? Drink plenty of fluids and give it some time. You'll recover.`

Reply:
I'm rewording that. On the second thought it didn't even sound well in my native language.